packages/react-ui/storybook/utils.js

```diff
@@ -0,0 +1,44 @@
+function addCommaRecognizer(p) {
+  return p.toString().replace(/,/g, '_commaInArray');
+}
+
+function makeValueTransformation(prop) {
+  if (Array.isArray(prop)) {
+    if (prop.every((p) => typeof p === 'object')) {
+      return `[\n  ${prop.map((p) => JSON.stringify(p))}\n]`;
+    }
+
+    if (prop.every(String)) {
```

Comment:
What about adding a specific case for number arrays?

For example for this:
![image](https://user-images.githubusercontent.com/458196/111361260-ed9e0900-868d-11eb-8fb7-7b8dc06c9298.png)